porch/controllers/workloadidentitybinding/pkg/controllers/workloadidentitybinding/controller.go

```diff
@@ -29,6 +29,11 @@ import (
 	"k8s.io/klog/v2"
 	ctrl "sigs.k8s.io/controller-runtime"
 	"sigs.k8s.io/controller-runtime/pkg/client"
+	"sigs.k8s.io/controller-runtime/pkg/controller/controllerutil"
+)
+
+const (
+	finalizerName = "config.porch.kpt.dev/finalizer"
```

Comment:
Actually, we should change this to something like `config.porch.kpt.dev/workload-identity`. We don't need the word "finalizer" in there - we already know it's a finalizer because that's the list it is in. However, we may have multiple controllers that all use finalizers - the user should be able to tell which controller needs to clear the finalizer by looking at the name.

Reply:
Yeah, I changed this to `config.porch.kpt.dev/workloadidentitybindings`.